src/amo/components/HomeHeroBanner/index.js

```diff
@@ -1,58 +1,91 @@
 /* @flow */
 import * as React from 'react';
 import { compose } from 'redux';
+import makeClassName from 'classnames';
 
 import translate from 'core/i18n/translate';
 import Hero from 'ui/components/Hero';
 import HeroSection from 'ui/components/HeroSection';
 import type { I18nType } from 'core/types/i18n';
+import { experiment } from 'core/components/Experiment';
 
 import './styles.scss';
 
 type Props = {|
   i18n: I18nType,
+  trackClick: () => void,
+  variant: string,
 |};
 
+export const AB_HOME_HERO_TEST_NAME = 'HOME_HERO';
+export const AB_HOME_HERO_VARIANT_A = 'ScaledDown';
+export const AB_HOME_HERO_VARIANT_B = 'Large';
+
 export class HomeHeroBannerBase extends React.Component<Props> {
   sections() {
-    const { i18n } = this.props;
-
+    const { i18n, trackClick } = this.props;
+    // TODO: make this into an array that you can map through.
+    // This might help eliminate manual prop set up / redundancy.
```

Comment:
Yep, doing this before your PR would reduce the diff of this one.

Reply:
oh? I thought it should be a separate PR..? Sure tho, I can update here :)

Reply:
Yes, that would be nice as a separate PR but still merged before this PR here :)

Reply:
new PR for this #6180 : )